beetsplug/subsonicupdate.py

```diff
@@ -90,18 +97,37 @@ def __format_url():
                 context_path = ''
             url = "http://{}:{}{}".format(host, port, context_path)
 
-        return url + '/rest/startScan.view'
+        return url + '/rest/{}'.format(endpoint)
+
+    def get_version(self):
```

Comment:
```suggestion
    def __get_version(self):
```

I believe we want to make this private.